cmd/kubeadm/app/util/version.go

```diff
@@ -72,46 +75,49 @@ func KubernetesReleaseVersion(version string) (string, error) {
 	}
 
 	// revalidate, if exact build from e.g. CI bucket requested.
-	ver = normalizedBuildVersion(versionLabel)
-	if len(ver) != 0 {
+	if ver, err := normalizeBuildVersion(versionLabel); err == nil {
```

Comment:
Considering the complexity of this function it would be great to comment with examples 
e.g.
```go
// if versionLabel is a semantic version, use it; valid formats include
//  - 1.13.0 or v1.13.0 (version with or without initial v)
//  - ci/1.13.0 or ci/v1.13.0 (with initial bucket prefix; valid prefix are release, ci, ci-cross)
```

Reply:
This comment wouldn't be correct because the bucket is supposed to be a part of `bucketURL` after it has been split by `splitVersion()`. I moved the prefixed examples upwards to `KubernetesReleaseVersion`'s description.